### ISSUE TITLE
How to follow redirect

### ISSUE THREAD

**Issue by cnotoham** [silkimen/cordova-plugin-advanced-http#111]
Hi Sefa,

I have been using this plugin for a while now. It's been great!
We now have a new requirement whereby I need to follow a redirect (302) when I do a get.
I am using version 1.7.1 of the plugin but it seems that the plugin does not support follow redirect.

The scenario is like this:
- get to url1
- get response 302 move temporarily to url2
- url2 do 302 to url3
- url3 do 302 to url1
- get 200

The plugin simply returns 200. But if I use chrome browser, it follows the redirect.

Are you able to help me please?

Thank you and Regards,
Cornelius
p.s I am using ionic

**Comment by silkimen** [comment]
Hi Cornelius,

unfortunately, I didn't understand your problem. When the plugin is returning 200 it has already followed the redirects, right? Otherwise it would return 302 or did I misunderstand?

Regards,
Sefa

**Comment by cnotoham** [comment]
Hi Sefa,

Let me rephrase my question.  Do you know if the version 1.7.1 of the plugin set the disableRedirect defaulted to true?

t seems to me the server that I am hitting returns me a 200 to some other page when it detects a 302 is disabled.

Thanks,
Cornelius

**Comment by silkimen** [comment]
> Do you know if the version 1.7.1 of the plugin set the disableRedirect defaulted to true?

No, "disableRedirect" never defaulted to true.

>It seems to me the server that I am hitting returns me a 200 to some other page when it detects a 302 is disabled.

Your server can't "detect" that transparent redirecting is turned off in your client. Your server just responds with code 302 and the client needs to decide whether it follows the redirect or not. There is no way your server can detect this in advance.

**Comment by cnotoham** [comment]
Yes, I can see the redirect url in the response.url and my issue is solved by manually redirecting.
Thanks for your help.

**Comment by jagpreetsingh83** [comment]
@cnotoham How did you performed manual redirect? Do you have a code sample?
